transaction/src/signing/secp256k1/private_key.rs

```diff
@@ -27,6 +27,16 @@ impl Secp256k1PrivateKey {
         self.0.secret_bytes().to_vec()
     }
 
+    pub fn to_hex(&self) -> String {
```

Comment:
We convert between key and hex in a few places, makes sense for the type type to have these methods IMO.